packages/core/examples/numericInputExample.tsx

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+
+import {
+    Classes,
+    Intent,
+    NumericInput,
+    Position,
+    Switch,
+} from "@blueprintjs/core";
+
+import BaseExample, { handleBooleanChange, handleNumberChange } from "./common/baseExample";
+import { IntentSelect } from "./common/intentSelect";
+
+const SAMPLE_LEFT_ICON = "variable";
+const SAMPLE_PLACEHOLDER = "Enter a number...";
+
+export interface INumericInputExampleState {
+
+    buttonPositionIndex?: number;
```

Comment:
maybe, but the current setup allows me to reuse `this.renderSelectMenu` here. okay if i leave as is?